test/wilson/dom_test.cljs

```diff
@@ -177,6 +178,20 @@
                 [:td "false"]
                 [:td "true"]]]]))))
 
+(deftest sorted-table-test
+  (let [rows [{:a 1 :b "B" :c -6}
+               {:a 4 :b "A" :c 2}
+               {:a 3 :b "D" :c 4}]
+        ks (d/prepare-keys [:a :b :c])
+        component (d/sorted-table ks rows)
+        [table-headers table-rows] (match component
+                                     [:table _
+                                      [:thead
+                                       [:tr & table-headers]]
+                                      [:tbody & table-rows]]
+                                     [table-headers table-rows])]
```

Comment:
Actually, scratch that - I forgot I am returning a function now so that match will not work in my case, d'oh. This solution works